### ISSUE TITLE
Icon request: Unreal Engine

### ISSUE THREAD

**Issue by maddisondesigns** [FortAwesome/Font-Awesome#20052]
### Use case

Would love to see an icon for Unreal Engine, thanks.

Unreal Engine is a world leading gaming engine developed by Epic Games. Epic Games also has a marketplace allowing developers and artists to sell 2D and 3D assets, sound effects, plugins, environments, characters etc.. Having an Unreal Engine logo would allow marketplace sellers to use it when linking to their marketplace profiles.

As a sidenote, you already have a logo for Unity, which is another leading Game Engine with a similar marketplace, so having a logo for Unreal Engine as well would be a huge benefit as many artists/developers sell assets for both engines.

I have attached an SVG of the logo.

### Example image

![unreal-engine-logo](https://github.com/FortAwesome/Font-Awesome/assets/2078245/765afcf3-5f03-46c1-9e56-93b8b11c77d4)

### Icon request checklist

- [X] The title starts with "Icon request: " and is followed by the requested icon name
- [X] This icon does not represent a brand
- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The request is for a concrete object, or I've included an example image
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/new#faqs)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

This request is a duplicate of #11420 , please add a 👍 reaction to that request.

You can [search for existing requests](https://github.com/FortAwesome/Font-Awesome/search?type=Issues) before opening new ones. If you searched before, maybe search did not work.

You may also be interested in taking a look at our [Requesting Icons FAQs](https://fontawesome.com/support#requesting-icons), they contain a lot of information about how requests work

Another useful resource is our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new)

Closing here

**Comment by maddisondesigns** [comment]
Thanks @tagliala. Will Do. I did perform a search before adding this request but nothing came up. Looking at the Title for #11420 I notice that it says `Icone Request` instead of `Icon Request`. It might be a good idea to change that Title since it doesn't return in the results when doing a search within all the Icon Requests.

I just noticed it also doesn't have the "new icon" label either.

<img width="892" alt="screenshot_1828" src="https://github.com/FortAwesome/Font-Awesome/assets/2078245/3481f031-60dc-4a56-9287-d6e3e6b13793">
